duckduckgo/duckduckgo-privacy-extension
# Add ignore regex for /software pages

**Reviewer:**
@MariagraziaAlastra 

/CC:
@jdorweiler 

## Description:
Prevents the Extension from opening the Post Install page on `duckduckgo.com/software/` urls

## Steps to test this PR:
**Automated Tests**: `npm run test` (I've added unit tests for the regex logic

**Manual Test**: 
- Uninstall extension from Firefox, and Chrome. For each, do this:
    1. Go to https://moollaza.duckduckgo.com/software/q-dir-portable?success=1
    2. You should see two ATB badges
    3. Click a badge and follow inline install
    4. After extension installs, we should **not** open https://duckduckgo.com/app in a new tab
    5. The same should still apply when installing the app from https://moollaza.duckduckgo.com/app

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
